scraper/src/meilisearch_helper.py

```diff
@@ -100,6 +102,7 @@ class MeiliSearchHelper:
 
     def __init__(self, host_url, api_key, index_uid, custom_settings):
         self.meilisearch_client = meilisearch.Client(host_url, api_key)
+        self.meilisearch_client.http.headers["User-Agent"] = qualified_version()
```

Comment:
Even though I know you can append the new `qualified_version()` into the current header value, I don't want that approach because it can introduce unwanted behaviors.

I would like to see that change in the constructor of Meilisearch Python as we have in the other SDKs because this constructor ability will be useful in the future in a possible Django integration for example.

Reply:
> Even though I know you can append the new qualified_version() into the current header value, I don't want that approach because it can introduce unwanted behaviors.

Yes, I agree. I'll have to publish a new release to do this. So wouldn't it be better to wait until next week for release week?